Runner/JobManager.py

```diff
@@ -0,0 +1,154 @@
+import azure.storage.blob as azureblob
+import azure.batch.batch_service_client as batch
+import azure.batch.batch_auth as batchauth
+import azure.batch.models as batchmodels
+import azext.batch as batch 
+import json
+import datetime
+import time
+import sys
+import traceback
+import asyncio
+import Utils
+import os
+from os import listdir
+from os.path import isfile, join
+from azure.storage.blob.models import BlobBlock, ContainerPermissions, ContentSettings
+import asyncio
+import datetime
+import random
+import types
+import functools  
+
+async def submit_job(batch_service_client, template, parameters):
+    job_json = batch_service_client.job.expand_template(template, parameters)
+    job = batch_service_client.job.jobparameter_from_json(job_json)
+    loop = asyncio.get_event_loop()
+    await loop.run_in_executor(None, functools.partial(batch_service_client.job.add, job))    
+
+
+class JobManager(object):
+
+    def __init__(self, template_file, pool_template_file, parameters_file, expected_output, application_licenses=None):
+        super(JobManager, self).__init__()
+        self.job_id = Utils.get_job_id(parameters_file)
+        self.pool_id = Utils.get_pool_id(parameters_file)
+        self.template_file = template_file
+        self.parameters_file = parameters_file
+        self.application_licenses = application_licenses
+        self.expected_output = expected_output
+        self.pool_template_file = pool_template_file
+        self.storage_info = None 
+        self.job_status = False, "Job hasn't started yet."
+
+    
+    def __str__(self):
+        return "job_id: {} pool_id:{} ".format(self.job_id, self.pool_id)
+
+    async def RunJob(self, batch_service_client):
```

Comment:
💡 📓 Recommendation: Python now support `type annotations` so that can help in specifying typed of input to the function and return types: for example:

```
async def monty(
   self,
   obj_1: str
   obj_2: some_type
) -> return_type :
   funct_body
   pass
```

Reply:
Thanks I'll keep that in mind